edu_edfi_airflow/callables/airflow_util.py

```diff
@@ -84,17 +79,32 @@ def get_config_endpoints(context) -> List[str]:
 
 
 def xcom_pull_template(
-    task_ids: str,
-    key: str = 'return_value'
+    task_ids: Union[str, 'Operator', List[str], List['Operator']],
```

Comment:
Overload XCom Pull Template with two changes:
* Pass a task_id or an operator directly
* Allow an optional prefix and suffix to be applied.